samples/system-test/detect.v1p4beta1.test.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright 2019, Google, LLC.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const path = require('path');
+const {Storage} = require('@google-cloud/storage');
+const execa = require('execa');
+const {assert} = require('chai');
+const uuid = require('uuid');
+
+const exec = async cmd => (await execa.shell(cmd)).stdout;
+const storage = new Storage();
+const bucketName = `nodejs-docs-samples-test-${uuid.v4()}`;
+const cmd = `node detect.v1p4beta1.js`;
+
+const files = [`pdf-ocr.pdf`, `landmark.jpg`].map(name => {
+  return {
+    name,
+    localPath: path.resolve(path.join(__dirname, `../resources/${name}`)),
+  };
+});
+
+describe(`detect v1 p4 beta1`, () => {
+  before(async () => {
+    const [bucket] = await storage.createBucket(bucketName);
+    await Promise.all(files.map(file => bucket.upload(file.localPath)));
+  });
+
+  after(async () => {
+    const bucket = storage.bucket(bucketName);
+    await bucket.deleteFiles({force: true});
```

Comment:
Why do we try to delete files twice?

Reply:
I mostly followed what I see it in other tests files such as detect.test.js. I will delete the 2nd one and see what happens.